src/Sulu/Bundle/SnippetBundle/Resources/public/js/components/webspace/settings/snippets/main.js

```diff
@@ -13,8 +13,8 @@ define(['config', 'app-config'], function(Config, AppConfig) {
 
     var defaults = {
         options: {
-            snippetTypesUrl: '/admin/api/snippet-types?defaults=true&webspace=<%= webspace %>',
-            snippetTypeDefaultUrl: '/admin/api/snippet-types/<%= type %>/default?webspace=<%= webspace %>',
+            snippetAreasUrl: '/admin/api/snippet-types/areas?webspace=<%= webspace %>',
```

Comment:
I would also prefer an own snippet area controller under `/admin/api/snippet-areas) as described in #3402.
Is it ok to move all default things (putDefaultAction, deleteDefaultAction, ..) out of the SnippetTypesController would be an internal bc break?